components/centraldashboard/public/components/notebook-default-card.css

```diff
@@ -0,0 +1,91 @@
+paper-card {
```

Comment:
I feel like some of these css rules are copied from the card-styles.js file, which you already import in your notebook-default-card.js file. You could be able to simplify this file a bit and remove duplicate code.